acceptance-tests/src/test/resources/uk/gov/di/test/features/Reset_password.feature

```diff
@@ -27,7 +27,7 @@ Feature: Reset password
     And the user logs out
 
 # REQUEST OTP TOO MANY TIMES DURING PASSWORD RESET --- AUT-1274
-  Scenario: A user is blocked when they request an email OTP more than 5 times during a password reset.
+  Scenario: A user is blocked when they enter an incorrect email OTP more than 6 times during a password reset.
```

Comment:
Not sure why this has been changed?

Reply:
This is because the test is entering an incorrect OTP

Reply:
Ah, apologies, i see it now. Couldn't tell that just from the 2 lines shown in this PR, had to look at the full test. Don't know how that's managed to be wrong for so long without us noticing!
Anyway, good spot. Can you make sure that everything in that test has been updated in line with the updates you've made. E.g. the commented out line above needs updating to reflect the change. Also, is the name of the data tag still correct? And i think it's still 'more than 5', ie. 6, not 'more than 6', ie.7?

Reply:
thats fine